### PR TITLE
CHI-1408: Dismiss Twilio warning notifications in E2E tests

### PR DESCRIPTION
**Pull request by stephenhand**
Primary reviewer: @mmythily 

## Description

* Added page functions to support dismissing Twilio warning messages and logging their text out
* Added artifacts helper to facilitate redirecting browser downloads to the test artefacts
* Added calls to check for, download & dismiss warnings at the end of tests and before trying to open the print view in the case list test

Note that this fix has been implemented on the Flex 2.0 branch. I didn't see any point in doing it on Flex 1.0 since we are about to merge and the Twilio UI might be different & require different selectors

### Checklist
- [X] Corresponding issue has been opened
- [X] New tests added
- N/A Strings are localized
- [X] Tested for chat contacts
- N/A Tested for call contacts

### Related Issues
Fixes CHI-1408

### Verification steps

* Throttle your browsers network bandwidth back so the 'degraded state' notification appears in Flex
* Run the E2E tests

Tests should pass and there should be a Twilio report in the test artifacts

### CODE REVIEW

File: .github/workflows/plugin-hrm-form-e2e-test.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: End to End Testing CI
 
 # Action 
-on: [pull_request]
+on: [pull_request, ]
```
Comment (mythilytm) [inline .github/workflows/plugin-hrm-form-e2e-test.yml:4]: Interesting workflow syntax

**Review by mythilytm — APPROVED** [review]
Looks and works great

File: .github/workflows/plugin-hrm-form-e2e-test.yml (hunk 1 above)
Comment (stephenhand) [reply .github/workflows/plugin-hrm-form-e2e-test.yml:4]: Yes that was a mistake corrected in the other PR you approved - I'll merge up to make sure they merge correctly

Comment (mythilytm) [reply .github/workflows/plugin-hrm-form-e2e-test.yml]: Ahh, my bad. 👍

Comment (stephenhand) [reply .github/workflows/plugin-hrm-form-e2e-test.yml]: No, well spotted - the bad 'twas mine

**Pull request merged**